utopia_tree.py

```diff
@@ -0,0 +1,38 @@
+#!/bin/python3
```

Comment:
Nice, you can also add file encoding e.g.
```python
#!/usr/bin/env python
# -*- coding: utf-8 -*-
```

Reply:
Comment added